### PR TITLE
topology1: Add support for sof-rn-rt5682-max98360, sof-rn-rt5682-rt1019

### PR DESCRIPTION
**Pull request by balakishorepati**
Add topology support for max98360,rt1019 speaker amplifiers
for amd renoir platform.

Signed-off-by: balapati <balakishore.pati@amd.com>

### CODE REVIEW

**Review by sunilkumardommati — APPROVED** [review]

**Review by bhiregou — APPROVED** [review]

File: tools/topology/topology1/sof-rn-rt5682-rt1019.m4 [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include ACP DSP configuration
+include(`platform/amd/acp.m4')
+
+# Playback pipeline 1 on PCM 0 using max 2 channels of s16le.
+# Schedule 96 frames per 2000us deadline on core 0 with priority 0
```
Comment (plbossart) [inline tools/topology/topology1/sof-rn-rt5682-rt1019.m4:22]: is 2ms required? IIRC all existing topologies assume a 1ms period.

File: tools/topology/topology1/sof-rn-rt5682-rt1019.m4 [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+# Capture pipeline 2 on PCM 0 using max 2 channels of s16le.
+PIPELINE_PCM_ADD(sof/pipe-passthrough-capture.m4,
+        2, 0, 2, s16le,
+        2000, 0, 0,
+        48000, 48000, 48000)
+
+# Capture DAI is ACPSP using 2 periods
+DAI_ADD(sof/pipe-dai-capture.m4, 2, ACPSP, 0, acp-headset-codec,
+PIPELINE_SINK_2, 2, s16le, 2000, 0, 0, SCHEDULE_TIME_DOMAIN_DMA)
```
Comment (plbossart) [inline tools/topology/topology1/sof-rn-rt5682-rt1019.m4:46]: add tabs or something to indent the code please.

File: tools/topology/topology1/sof-rn-rt5682-rt1019.m4 [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+PCM_DUPLEX_ADD(I2SSP, 0, PIPELINE_PCM_1, PIPELINE_PCM_2)
+
+# Capture pipeline 3 on PCM 1 using max 2 channels of s32le.
+PIPELINE_PCM_ADD(sof/pipe-passthrough-capture.m4,
+        3, 1, 2, s32le,
+        2000, 0, 0,
+        16000, 16000, 16000)
+
+DAI_ADD(sof/pipe-dai-capture.m4, 3, ACPDMIC, 0, acp-dmic-codec,
+PIPELINE_SINK_3, 2, s32le, 2000, 0, 0, SCHEDULE_TIME_DOMAIN_DMA)
```
Comment (plbossart) [inline tools/topology/topology1/sof-rn-rt5682-rt1019.m4:58]: indentation

File: tools/topology/topology1/sof-rn-rt5682-max98360.m4 [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+# Capture pipeline 2 on PCM 0 using max 2 channels of s16le.
+PIPELINE_PCM_ADD(sof/pipe-passthrough-capture.m4,
+        2, 0, 2, s16le,
+        2000, 0, 0,
+        48000, 48000, 48000)
+
+# Capture DAI is ACPSP using 2 periods
+DAI_ADD(sof/pipe-dai-capture.m4, 2, ACPSP, 0, acp-headset-codec,
+PIPELINE_SINK_2, 2, s16le, 2000, 0, 0, SCHEDULE_TIME_DOMAIN_DMA)
```
Comment (plbossart) [inline tools/topology/topology1/sof-rn-rt5682-max98360.m4:46]: indentation

File: tools/topology/topology1/sof-rn-rt5682-max98360.m4 [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+PCM_DUPLEX_ADD(I2SSP, 0, PIPELINE_PCM_1, PIPELINE_PCM_2)
+
+# Capture pipeline 3 on PCM 1 using max 2 channels of s32le.
+PIPELINE_PCM_ADD(sof/pipe-passthrough-capture.m4,
+        3, 1, 2, s32le,
+        2000, 0, 0,
+        16000, 16000, 16000)
+
+DAI_ADD(sof/pipe-dai-capture.m4, 3, ACPDMIC, 0, acp-dmic-codec,
+PIPELINE_SINK_3, 2, s32le, 2000, 0, 0, SCHEDULE_TIME_DOMAIN_DMA)
```
Comment (plbossart) [inline tools/topology/topology1/sof-rn-rt5682-max98360.m4:58]: indentation

**Review by plbossart — CHANGES_REQUESTED** [review]
basic indentation issues plus question on 2ms period.

File: tools/topology/topology1/sof-rn-rt5682-rt1019.m4 (hunk 1 above)
Comment (bhiregou) [reply tools/topology/topology1/sof-rn-rt5682-rt1019.m4:22]: These are just sample topologies to show case usage of AMD platforms, nothing specific about 2ms.

File: tools/topology/topology1/sof-rn-rt5682-rt1019.m4 (hunk 2 above)
Comment (balakishorepati) [reply tools/topology/topology1/sof-rn-rt5682-rt1019.m4]: Add

> add tabs or something to indent the code please.

Addressed the review comment.

File: tools/topology/topology1/sof-rn-rt5682-rt1019.m4 (hunk 3 above)
Comment (balakishorepati) [reply tools/topology/topology1/sof-rn-rt5682-rt1019.m4]: Addressed the review comment.

File: tools/topology/topology1/sof-rn-rt5682-max98360.m4 (hunk 4 above)
Comment (balakishorepati) [reply tools/topology/topology1/sof-rn-rt5682-max98360.m4]: Addressed the review comment.

File: tools/topology/topology1/sof-rn-rt5682-max98360.m4 (hunk 5 above)
Comment (balakishorepati) [reply tools/topology/topology1/sof-rn-rt5682-max98360.m4]: Addressed the review comment.

**Review by lgirdwood — APPROVED** [review]
Indentation looks correct now.

**Pull request merged**